CHANGES

```diff
@@ -46,19 +47,31 @@ Tactics
 - In "Tactic Notation" or "TACTIC EXTEND", entry "constr_with_bindings"
   now uses type classes and rejects terms with unresolved holes, like
   entry "constr" does. To get the former behavior use
```

Comment:
Well, I am not the author of these lines but I don't think so. As I understand it, what is meant is, on the contrary, that constr rejects terms with unresolved holes as well. Can someone confirm? @herbelin @ppedrot